pkg/infrastructure/gcp/clusterapi/clusterapi.go

```diff
@@ -33,6 +33,16 @@ func (p Provider) Name() string {
 // GCP resources that are not created by CAPG (and are required for other stages of the install) are
 // created here using the gcp sdk.
 func (p Provider) PreProvision(ctx context.Context, in clusterapi.PreProvisionInput) error {
+	// Create ServiceAccount which will be used for machines
+	projectID := in.InstallConfig.Config.Platform.GCP.ProjectID
+	saID, err := CreateServiceAccount(ctx, in.InfraID, projectID, "master")
```

Comment:
we will also need to create [the service accounts for the compute nodes](https://github.com/openshift/installer/blob/master/data/data/gcp/cluster/iam/main.tf) (because they were being created by terraform)